scripts/proto-generate.sh

```diff
@@ -14,4 +14,5 @@ exec protoc \
   --js_out=import_style=commonjs,binary:src/proto/js \
   --ts_out="grpc_js:${script_dir}/../src/proto/js" \
   --grpc_out="grpc_js:${script_dir}/../src/proto/js" \
-  "${script_dir}/../src/proto/schemas/"*.proto
+  "${script_dir}/../src/proto/schemas/"**/*.proto \
```

Comment:
Does this already handle globbing all the proto files? Then the second line of `"${script_dir}/../src/proto/schemas/"*.proto \` should be unnecessary.

Reply:
If I do just the line with the glob then it it only generates sub-directory.

Reply:
Try: https://unix.stackexchange.com/a/49917/56970

Reply:
Update the bash script accordingly.

Reply:
That fixed it. it was enabled in the console but disabled when running the script. adding `shopt -qs globstar` fixed it.